3-With-Intune MAM/README.md

```diff
@@ -0,0 +1,53 @@
+# Integrate Microsoft Identity, the Microsoft Authenticator App (broker), and Microsoft Intune MAM SDK into a Xamarin Forms app using MSAL.NET.
+
+This sample demonstrates how to access [Intune Mobile App Management (MAM)](https://docs.microsoft.com/en-us/mem/intune/apps/app-management) protected resources using MSAL.NET
+
+## Overview
+When a user attempts to access a MAM protected resource and if the device is not compliant, MSAL.NET will throw ```IntuneAppProtectionPolicyRequiredException```. The common code catches the exception and requests Intune MAM SDK to make the app compliant. Intune MAM SDK usage is platform specific. So this exception handling behavior is executed using Inversion of Control pattern, i.e. using a common interface with platform specific implementations.  
+
+After the device becomes compliant, the App makes silent token request to obtain the token.  
+
+The sample has a code tour that shows the workflow of the app.  You can open the source code in [VS Code](https://code.visualstudio.com/). Get the [CodeTour Extension](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) and run the steps. It will walk you through the code flow.  
+
+**Note:** The sample showcases only the scenario when user completes the compliance request. Handling of other scenarios when user does not satisfy the compliance request is up to the business requirements of the application.
+
+## How To Run this Sample
+
+### Prerequisites
+To run this sample you will need:
+- [Visual Studio 2019](https://aka.ms/vsdownload) with the **Mobile development with .NET** [workload](https://www.visualstudio.com/vs/visual-studio-workloads/):
+- An Internet connection
+- At least one of the following accounts:
+  - A Microsoft Account in Premium tier- you can get a free account by visiting [https://www.microsoft.com/en-us/outlook-com/](https://www.microsoft.com/en-us/outlook-com/).
+  - An Azure AD account - you can get a free trial Office 365 account by visiting [https://products.office.com/en-us/try](https://products.office.com/en-us/try).
+- Setup [Xamarin.iOS for Visual Studio](https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/windows/introduction-to-xamarin-ios-for-visual-studio) (if you want to run the iOS app)
+
+### Step 1: Setup Azure Active Directory and Intune Portal
+The app requires setup in Azure Active Directory to obtain your parameters. The setup instructions can be found [here](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Steps-to-create-config-for-MAM-(Conditional-access))

```

Comment:
It is best to put all of the setup steps here even if they are redundant. Wiki links in MSAL are often changed/are moved to MS Docs and there is a high probability that the wiki link will not work in the future.

Reply:
Good point. I will suggest that this becomes 3 step tutorial and be handled by CxP team.

Reply:
Updated